app/decorators/decidim/assemblies_decorator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+#
+# This decorator adds required associations between Decidim::ParticipatoryProcess and Departaments.
+#
+#
+require_dependency 'decidim/assembly'
+Decidim::Assembly.class_eval do
+  belongs_to  :areas,
```

Comment:
Why do we need this belongs_to as we already have area attribute @spuigmitjadxc  ?